components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -193,6 +196,21 @@ const PrepopulatedEngine startpage = {
     PREPOPULATED_ENGINE_ID_STARTPAGE,
 };
 
+const PrepopulatedEngine brave_yandex =
+  ModifyEngineParams(yandex_com,
+    L"Yandex",
+    NULL,
+    "https://yandex.ru/search/?clid="
+#if defined(OS_ANDROID)
+    "2423859"
+#else
+    "2353835"
+#endif
+    "&text={searchTerms}",
+    "https://suggest.yandex.com/suggest-ff.cgi?"
+    "part={searchTerms}&uil=en&v=3&sn=5&srv=brave_desktop",
```

Comment:
I'm not 100% sure what exactly does `uil`, maybe we should set `uil=ru` (given that the whole thing is for russian-speaking locations)?

Reply:
not all locations are Russian speaking, Turkey, Armenia for example

Reply:
we don't enable it for Turkey, do we?
I'm pretty sure that in any of these countries we would have much more searches in Russian than in English (not counting local languages). Anyway, I have no clue what this parameter is about, and after doing a couple of queries I don't see any difference

Reply:
right, there's `Turkmenistan` not Turkey :D
i was looking at list from here https://github.com/brave/security/issues/257#issue-738384467

Reply:
For sure, I need to change `suggest.yandex.com`, which should instead be `suggest.yandex.ru`. @iccub what does this URL look like on iOS side? does it have `uil=en` or `uil=ru`?

Reply:
```
<Url type="application/x-suggestions+json" method="GET" template="https://suggest.yandex.net/suggest-ff.cgi">
  <Param name="part" value="{searchTerms}"/>
</Url>
```

we only get 1 param in,
same on FF-iOS
https://github.com/mozilla-mobile/firefox-ios/blob/76faae8c6c94e22c8fbc0de72e2d06f615738e2c/Client/Assets/Search/SearchPlugins/yandex-ru.xml

Reply:
I removed `uil` altogether after confirming other examples which left it out 👍 Updated to be `suggest.yandex.ru`

Reply:
btw, maybe we could alsoo double-check the SERP domain? Basically Yandex has local domains for some countries (e.g. yandex.kz or yandex.az) - probably we can utilise them

Reply:
We definitely can utilize those - the person helping with integration has advised to use the .ru TLD for this integration